### ISSUE TITLE
IllustratedMessage: graphic is always grey in HCM

### ISSUE THREAD

**Issue by jnurthen** [adobe/react-spectrum#4143]
# 🐛 Bug Report 

The graphic in illustratedMessage is hardcoded to --spectrum-global-color-gray-500
## 🤔 Expected Behavior

It should be the same as the text color in High Contrast mode
## 😯 Current Behavior

https://react-spectrum.adobe.com/react-spectrum/IllustratedMessage.html
Run in High Contrast mode - note that the graphics are always grey

## 💁 Possible Solution

provide an override for https://github.com/adobe/react-spectrum/blob/cb1fc23c8f9efce828f774858aaf3a06a19b49a2/packages/%40adobe/spectrum-css-temp/components/illustratedmessage/skin.css#L14 in High Contrast mode so color is CanvasText

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3
| Browser          | Edge
| Operating System | Windows High Contast mode

**Comment by devongovett** [comment]
FYI, many of the WHCM issues that have been raised in your audit (including this one) were found as part of our initial testing after it was first merged: https://github.com/adobe/react-spectrum/issues/2949. Might be worth going through those and deduping.
